libcxx/include/__atomic/atomic.h

```diff
@@ -467,13 +467,13 @@ _LIBCPP_HIDE_FROM_ABI bool atomic_is_lock_free(const atomic<_Tp>* __o) _NOEXCEPT
 // atomic_init
 
 template <class _Tp>
-_LIBCPP_DEPRECATED_IN_CXX20 _LIBCPP_HIDE_FROM_ABI void
+[[__libcpp_deprecated_in_cxx20()]] _LIBCPP_HIDE_FROM_ABI void
```

Comment:
Are the empty parenthesis required?

Reply:
They are required if we want to be able to add a message. I think it'd be better this way to encourage people to add a message, but it's not required for the main change. If that's the hangup I can revert these changes.

Reply:
I think it's good to be able to provide a message. However there is not always a replacement in the Standard. So I rather have two macros one for that provides a message and one that doesn't. That avoids the `()` which looks bad to me.